aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1444,6 +1444,7 @@ impl AptosVM {
         mut expected_modules: BTreeSet<String>,
         allowed_deps: Option<BTreeMap<AccountAddress, BTreeSet<String>>>,
     ) -> VMResult<()> {
+        self.check_unstable_bytecode(modules)?;
```

Comment:
nit: `filter_unstable_bytecode`, or something similar. This terminoligy is a bit confusing.

Reply:
done